api/data_explorer/controllers/facets_controller.py

```diff
@@ -18,9 +15,6 @@ def facets_get():
         List of Facets.
     """
     search = DatasetFacetedSearch()
-    search.using = Elasticsearch(current_app.config['ELASTICSEARCH_URL'])
```

Comment:
I noticed this PR breaks the UI. Are there references to this function that need to be updated to the facets_get in dataset_faceted_search?